clients/admin-ui/src/features/config-wizard/AddSystemForm.tsx

```diff
@@ -0,0 +1,133 @@
+import {
+  Accordion,
+  AccordionButton,
+  AccordionItem,
+  AccordionPanel,
+  chakra,
+  FormControl,
+  Heading,
+  HStack,
+  IconButton,
+  Stack,
+  Text,
+  Tooltip,
+} from "@fidesui/react";
+import type { NextPage } from "next";
+import React from "react";
+
+import {
+  AWSLogoIcon,
+  ManualSetupIcon,
+  OktaLogoIcon,
+  QuestionIcon,
+} from "~/features/common/Icon";
+
+const AddSystemForm: NextPage<{
+  handleChangeStep: Function;
+}> = ({ handleChangeStep }) => (
+  <chakra.form
```

Comment:
what is `chakra.form`? I saw you used it a few times now, but curious about what it actually is and when to use it?

Reply:
https://chakra-ui.com/docs/styled-system/features/chakra-factory

Reply:
ohhh very cool!